arrow/src/main/java/org/apache/iceberg/arrow/vectorized/VectorizedArrowReader.java

```diff
@@ -517,5 +517,32 @@ public void setBatchSize(int batchSize) {
     }
   }
 
+  /**
+   * A Dummy Vector Reader which doesn't actually read files, instead it returns a dummy
```

Comment:
Just a little note here, I think you don't have to say "dummy" twice. 

Small suggestion?
"A Vector Reader that returns a IsDeleted Vector Holder which returns true when a given row is deleted when read"?